src/django_components/cache.py

```diff
@@ -29,18 +29,19 @@ def get_template_cache() -> LRUCache:
 
 
 def get_component_media_cache() -> BaseCache:
+    if app_settings.CACHE is not None:
```

Comment:
Another bug I found - this relates to the [cache for storing the JS /  CSS scripts](https://github.com/django-components/django-components/pull/946). 

Previously the logic was that we checked only the first time whether to use user-provided cache or a default one, and then reused that for eternity.

Now, we check each time we want to access the cache, which one to use.

This was causing errors when all tests were run together.